python/whylogs/api/writer/writer.py

```diff
@@ -1,6 +1,8 @@
 import os
 from abc import ABC, abstractmethod
-from typing import Any, Optional, TypeVar
+from typing import Any, Optional, TypeVar, Union
+
+import requests  # type: ignore
```

Comment:
we can't import this package as a top level import in Writers, it is not a dependency of whylogs and shouldn't be. 'requests' should be specific to the [whylabs] extra and any reference needs to guard against cases where users don't have it (see PIL or pandas import patterns)

Reply:
Got it. So only in the whylabs one then. :) I can change this to an any type for the abstract return which will be able to keep this from being here